db/migrations/20240612112949_add_switch_delay_and_video_pause_delay_to_sync_group.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/*
+ * Copyright (C) 2024 Xibo Signage Ltd
+ *
+ * Xibo - Digital Signage - https://xibosignage.com
+ *
+ * This file is part of Xibo.
+ *
+ * Xibo is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * any later version.
+ *
+ * Xibo is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with Xibo.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+use Phinx\Migration\AbstractMigration;
+
+class AddSwitchDelayAndVideoPauseDelayToSyncGroup extends AbstractMigration
```

Comment:
These PHPCS errors can both be addressed please. For the namespace issue we can suppress it in a class comment

```
@phpcs:disable PSR1.Classes.ClassDeclaration.MissingNamespace
```

You can run PHPCS locally which would highlight both of these ahead of creating a PR.

Reply:
Finally, PHPCS errors has also been addressed on this [commit](https://github.com/xibosignage/xibo-cms/pull/2564/commits/d2bf44bfdc0bae292e60291730cf72b9b009b411). Thank you again for the review!